pkg/plugins/manager/manager_test.go

```diff
@@ -282,6 +282,32 @@ func TestPluginManager_Init(t *testing.T) {
 		assert.Nil(t, pm.plugins[("test")])
 	})
 
+	t.Run("With back-end plugin with valid v2 private signature (plugin root URL ignores trailing slash)", func(t *testing.T) {
+		origAppURL := setting.AppUrl
+		t.Cleanup(func() {
+			setting.AppUrl = origAppURL
+		})
+		setting.AppUrl = "http://localhost:3000/grafana/"
```

Comment:
This should perhaps be split up in `setting.AppUrl` and `setting.AppSubUrl` to be more close to reality? Based on https://github.com/grafana/grafana/blob/1f5a28ec63c3a6d0748aa58c08f4b25e166c0ccd/pkg/setting/setting.go#L1364-L1371